### ISSUE TITLE
Settings Server Prefix default to /usr/local/bin/jackd?

### ISSUE THREAD

**Issue by wholenotes** [rncbc/qjackctl#102]
Could the Server Prefix field default to /usr/local/bin/jackd, which is where the jack installer places it?  Would be an enhancement in usability.

**Comment by rncbc** [comment]
nope. that is seldom the default server path or prefix; it is only that so when you build and install jackd from source (eg. configure --prefix=/usr/local is usually the default on that case).

you ought to add `/usr/local/bin` to your PATH environment variable and stay with just `jackd` as server prefix on qjackctl advanced parameters (which you are warned to not touching unless you know what you're doing).

cheers

**Comment by wholenotes** [comment]
I'm using the version of JACK on the CCRMA site (https://ccrma.stanford.edu/software/jacktrip/osx/index.html), which is compatible with my OS X installation -- the latest unofficial JACK installer didn't work for me on Catalina.  The older installer on the CCRMA site places the binary in /usr/local/bin/jackd.  I haven't paid attention to where the latest JACK installer places the binary.  Fernando Lopez-Lezcano and I are trying to reduce the number of installation steps users of JackTrip have to take, especially in the terminal.  With the pandemic, many users who have never used the command line are starting to use JackTrip.  While changing the PATH variable may be easy for those who have been in the digital audio space for a number of years, there's now a different class of users we're trying to help.  Not a big deal.
